packages/dashmate/test/e2e/lib/constants/statusOutput.js

```diff
@@ -0,0 +1,10 @@
+const preSyncedStatus = {
```

Comment:
Is it used somewhere?

Reply:
Yes, it's used in assertions for both, masternode (e.g. core status) and upcoming fullnode tests (later for mainnet as well). I'll complete that file about missing statuses for synced nodes and rename everything in the future.